anyway/models.py

```diff
@@ -47,13 +46,23 @@
     datefmt="%Y-%m-%d %H:%M:%S",
 )
 
-roles_users = Table(
-    "roles_users",
+users_to_roles2 = Table(
```

Comment:
Why is this table different in the way it is defined, and in its name style from the other tables? (why isn't it a class?)

Reply:
The only place this table is being accessed directly is to create the relationship with the user class - to allow indirect access:
![image](https://user-images.githubusercontent.com/58947331/116309231-2fec5700-a7b1-11eb-97d6-c035816966c5.png)
and the `secondary` argument need object of type `Table`.
Anyone that what to access the data in this table need to do it by accessing the user class.

That's way the definition of this table is different.